sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/ServiceBusMessageBatch.cs

```diff
@@ -51,6 +51,27 @@ public sealed class ServiceBusMessageBatch : IDisposable
         ///
         private readonly TransportMessageBatch _innerBatch;
 
+        /// <summary>
+        ///   The entity path of the Service Bus entity that this batch is associated with. To be
+        ///   used during instrumentation.
+        /// </summary>
+        ///
+        private readonly string _entityPath;
+
+        /// <summary>
+        ///   The fully qualified Service Bus namespace that the batch is associated with. To be
+        ///   used during instrumentation.
+        /// </summary>
+        ///
+        private readonly string _fullyQualifiedNamespace;
+
+        /// <summary>
+        ///   The list of diagnostic identifiers of messages added to this batch.  To be used during
+        ///   instrumentation.
+        /// </summary>
+        ///
+        private List<string> _messageDiagnosticIdentifiers { get; } = new List<string>();
```

Comment:
Should this be `readonly`?

Reply:
I have a getter that returns a read only list defined [here](https://github.com/Azure/azure-sdk-for-net/blob/9e1632fcd59960b1dcbe90ad1079e14fdf609465/sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/ServiceBusMessageBatch.cs#L186). Would it preferable to declare it `readonly` instead?